### DISCUSSION TITLE
@error stops wire:ignore if it's placed before the ignore

### DISCUSSION THREAD

**Discussion by Roywcm** [livewire/livewire#5042] [7. Bugs (archived)]
I want to place an error validation message above a field which should be ignored by ```wire:ignore```
For some reason, when I wrap the error message in html, it stops ignoring the underlaying div. 
So when I change the value, the updated value will also be updated in the ignored div.

Happens in Livewire v2.10.5.

Check this playground: https://laravelplayground.com/#/snippets/36c8c353-564f-44c8-ab31-bebda07c2b3c 

This is my code
```
<div>
   <form wire:submit.prevent="save">
        {{-- this works: @error('value') {{ $message }} @enderror --}}
        {{-- this not --}}
        @error('value') <span class="error">{{ $message }}</span> @enderror

        <div wire:ignore>
          Initial value: {{ $value }}
        </div>

        <input type="text" wire:model="value">

        <button>Submit</button>
   </form>
</div>
```

```
class test extends Component
{
    public $value = 'initial';

    public function save()
    {
        $this->validate([
            'value' => 'max:2'
        ]);
    }

    public function render()
    {
        return view('view');
    }
}
```

**Comment by Martleby** [comment]
Yeah, this can happen. In the troubleshooting section of the docs it recommends wrapping the @error @enderror directive block in a div or other tag.

https://laravel-livewire.com/docs/2.x/troubleshooting

HTH,
Martin.

**Reply by joshhanley** [reply]
@Roywcm yep I agree with @Martleby, you should wrap your `@error` block in a div or something 🙂

**Reply by SourovCodes** [reply]
Thanks man!
